velox/exec/Operator.h

```diff
@@ -422,6 +424,10 @@ class Operator : public BaseRuntimeStatWriter {
     return stats_;
   }
 
+  virtual uint64_t getBackgroundCpuTimeMs() const {
```

Comment:
Can you comment what does background cpu time mean for an operator? Thanks!
```
s/getBackgroundCpuTimeMs/backgroundCpuTimeMs/
```
Add test for this?

Reply:
Done